node/CHANGELOG.md

```diff
@@ -12,28 +12,25 @@ All notable changes to this project will be documented in this file.  The format
 
 ## [Unreleased]
 
-
-## [1.4.2] - 2021-11-11
-
-### Changed
-* There are now less false warnings/errors regarding dropped responders or closed channels during a shutdown, where they are expected and harmless.
-* Execution transforms are ordered by insertion order.
-
-### Removed
-* The config option `consensus.highway.unit_hashes_folder` has been removed.
-
-### Fixed
-* The block proposer component now retains pending deploys and transfers across a restart.
-
-
-## [1.4.0] - 2021-10-04
-
 ### Added
 * Add new event to the main SSE server stream accessed via `<IP:Port>/events/main` which emits hashes of expired deploys.
-* Add new event to the main SSE server stream across all endpoints `<IP:PORT>/events/*` which emits a shutdown event when the node shuts down.
+* Add new event to the main SSE server stream across all endpoints `<IP:Port>/events/*` which emits a shutdown event when the node shuts down.
+* Introducing fast-syncing to join the network, avoiding the need to execute every block to catch up.
+* Added `archival_sync` to `[node]` config section, along with archival syncing capabilities
 
 ### Changed
 * `enable_manual_sync` configuration parameter defaults to `true`.
+* Major rewrite of the contract runtime component.
```

Comment:
Not sure if that's still true?

Reply:
I browsed the changes in the PR when this line appeared (https://github.com/casper-network/casper-node/pull/1669/files) and yes, some of the changes are no longer true for the `fast-sync` (in particular the `chainspec_loader.rs` and `era_supervisor.rs`). But still, some of the changes remain on `dev`. So I guess it's a 50/50 :thinking: